python-sdk/tests/test_client.py

```
@@ -156,12 +156,18 @@ def test_tracer_provider_reinitialized_on_api_key_change():
 
 def test_skip_open_telemetry_setup_property():
     """Test that skip_open_telemetry_setup property returns the correct value."""
+    # Reset singleton to start fresh
+    Client._reset_instance()
+    
     client = make_client(api_key="test-key", skip_open_telemetry_setup=True)
     assert client.skip_open_telemetry_setup is True
```

Ruff (0.12.2)

Use of `assert` detected

(S101)